.eslintrc.json

```diff
@@ -13,6 +13,14 @@
           {
             "selector": "MemberExpression[object.name='console'][property.name=/^(log|warn|error|info|trace)$/]",
             "message": "Don't log into the console directly. Use one of the functions from the log.ts file (logInfo, logWarning, logError) to ensure provided --logLevel is respected."
+          }, 
+          {
+            "selector": "CallExpression[callee.name='createManagementClient']",
+            "message": "Use provided create client in src/utils/client.ts"
```

Comment:
What about a more specific `Use the createClient function from src/utils/client.ts instead.`?